go/ephemeral/lib.go

```diff
@@ -11,21 +11,23 @@ import (
 	"github.com/keybase/client/go/erasablekv"
 	"github.com/keybase/client/go/libkb"
 	"github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/clockwork"
 )
 
 const SkipKeygenNilMerkleRoot = "Skipping key generation, unable to fetch merkle root"
 
-const cacheEntryLifetimeSecs = 60 * 5 // 5 minutes
+const cacheEntryLifetime = time.Minute * 5
```

Comment:
Just confirming the intention: This is going from seconds (our previous units) to nanoseconds (the Go std lib units)?

Reply:
yes since we're using `time.Time` calculations